### PR TITLE
Feature/api calls

### LINKED ISSUE #7: Write network request for search functionality
- Make this a drop-down for all supported cities from the API?
- Write functionality to locate the nearest supported city?

- We ended up going with a 2nd API that will get the latitude and longitude from a query (can be zip, city, county, address) and using that to fetch the nearest city data from the air quality API.

### PR DESCRIPTION
**Pull request by ErinUntermeyer**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- Added helper folder with apiCalls.js file
- Wrote network requests to get coordinates from a query and get location data from coordinates.
- Wrote method getTop5Cities in App to make the fetch call to display top 5 cities. It's currently returning a promise. A little tweak and it will work just fine!
- Closes #7 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [x] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:

# Message for reviewer:
- This will need a bit of tweaking to get the top 5 cities to display but the API calls needed are working!

# Checklist:
- [x] My code has no unused/commented out code
- [ ] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by leighlars — APPROVED** [review]
Great work taking the lead on the API calls. I'm sure the bug is minute. You're a rockstar

File: src/helpers/apiCalls.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	if (response.ok) {
+		const data = await response.json()
+		const coordinates = [data.data[0].latitude, data.data[0].longitude]
+		return getLocationData(parseInt(coordinates[0].toFixed(2)), parseInt(coordinates[1].toFixed(2)))
+	} else {
+		throw new Error(response.statusText)
+	}
+}
+
+export const getLocationData = async (lat: number, long: number) => {
```
Comment (leighlars) [inline src/helpers/apiCalls.tsx:13]: At least it's fetching some of it, meaning it's a small bug

**Review by leighlars — APPROVED** [review]

File: src/helpers/apiCalls.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+export const getCoordinates = async (query: string) => {
+	const response = await fetch(`http://api.positionstack.com/v1/forward?access_key=e17943cbd88c595c58c3c6ae1840fc33&query=${query}`)
```
Comment (JoshSevy) [inline src/helpers/apiCalls.tsx:2]: Great Job! I know its not fully there but this is a big chunk of what we need to get done. Thank you!

File: src/helpers/apiCalls.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+		const data = await response.json()
+		const coordinates = [data.data[0].latitude, data.data[0].longitude]
+		return getLocationData(parseInt(coordinates[0].toFixed(2)), parseInt(coordinates[1].toFixed(2)))
+	} else {
+		throw new Error(response.statusText)
+	}
+}
+
+export const getLocationData = async (lat: number, long: number) => {
+	const response = await fetch(`http://api.airvisual.com/v2/nearest_city?lat=${lat}&lon=${long}&key=068b5953-ff12-4969-bf21-787d07fe61bf`)
```
Comment (JoshSevy) [inline src/helpers/apiCalls.tsx:14]: I think we will be able to add in some params for this fetch call, this looks great.

**Pull request merged**